AgContext.yaml

```diff
@@ -0,0 +1,149 @@
+
+agcontext_name:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      Agricultural context name
+      string. Required.
```

Comment:
I'm not sure that we need a name. If it's required, I think its description should give some sense of what it's required for: will it be used as some rigid identifier.

Reply:
I realise now that this is just replicating the dataset/collection name. I don't think it is needed; as multiple agcontexts can exist within a dataset/collection.